### ISSUE TITLE
introduce Ableton bank parameter names

### ISSUE THREAD

**Issue by markusschloesser** [markusschloesser/MackieC4_P3#60]
Ableton devices have bank names.
Having those bank names shown on the C4 when banking params, would be helpful. Especially as the param names are abbreviated to 6 characters and devices like wavetable have shitloads of params (8 banks) `'Oscillator 1', 'Oscillator 2', 'Filter 1', 'Filter 2', 'Amp Envelope', 'Envelope 2/3', 'LFO 1/2', 'Global'`. 
1. Show current bank name
2. Show next and previous bank names 
 
Problem:
Ableton bank names consist of 8 params, a bank being ONE page.
Our "pages" consist of 24 params, meaning we would need to concatenate bank names.
==> might not be worth it, but for maybe later

**Comment by BilldarBagdar** [comment]
This will likely be related to more deeply linking the display text segments to encoders and will involved "aggregating" the 8 display segments over each row of encoders into "encoder row" objects or something...

Are you talking about showing the bank names on the top row of each of the 24 encoders per "script bank" in device mode?   

If so, we have 3 values we want to show on two lines of LCD,  The parameter name on top and the value on bottom, plus now the bank name on top.  The C4 commander app handles that by implementing "push & hold" on the encoder buttons to manually toggle the lower text state between showing "parameter name" or "parameter value" leaving the top for "bank text" or whatever.  Whenever actively turning a knob, lower text shows the values, but then after the value has been stable a couple of seconds, the display goes back to showing the parameter name if you have it toggled that way.

**Comment by markusschloesser** [comment]
> Are you talking about showing the bank names on the top row of each of the 24 encoders per "script bank" in device mode?

No, my idea was to use the top/angled display for that. We usually have some space there, and just for orientation reasons it might be helpful to what kind of bank a param belongs.
But imho low prio, especially as only Ableton Devices (and m4l) have bank names, vst plug-ins not (iirc)
